src/NServiceBus.Core/DelayedDelivery/DelayedDelivery.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace NServiceBus.Features
+{
+    using System;
+    using NServiceBus.CircuitBreakers;
+    using NServiceBus.Config;
+    using NServiceBus.DelayedDelivery;
+    using NServiceBus.DelayedDelivery.TimeoutManager;
+    using NServiceBus.DeliveryConstraints;
+    using NServiceBus.Pipeline;
+    using NServiceBus.Timeout.Core;
+    using NServiceBus.TransportDispatch;
+    using NServiceBus.Transports;
+
+    /// <summary>
+    /// Allows to delay sent messages.
+    /// </summary>
+    public class DelayedDelivery : Feature
+    {
+        internal DelayedDelivery()
+        {
+            EnableByDefault();
+            //TM
+            Defaults(s => s.SetDefault("TimeToWaitBeforeTriggeringCriticalErrorForTimeoutPersisterReceiver", TimeSpan.FromSeconds(2)));
+
+            Prerequisite(c => c.DoesTransportSupportConstraint<DelayedDeliveryConstraint>()
```

Comment:
The prerequisite check is using logical `AND` when there are multiple prerequisites. I can convert that to `AND` but I think it would be much less readable.